### ISSUE TITLE
mysql position table

### ISSUE THREAD

**Issue by cispune** [tananaev/traccar#1574]
i don't have data for devices in position table, yet on web panel it is showing data how?

**Comment by tananaev** [comment]
I guess it uses some other database. Possibly default H2.

**Comment by cispune** [comment]
i changed default database with sql, now data is updating in new sql tables, initially i added some data into tables through device tested and after that i deleted tables data and refreshed web ui but device still showing previous data how ?

**Comment by tananaev** [comment]
I guess Traccar is still using old database. Did you restart the service after changing the config?

**Comment by cispune** [comment]
yes restarted the service

On Fri, Dec 11, 2015 at 1:52 AM, Anton Tananaev <notifications@github.com>
wrote:

> I guess Traccar is still using old database. Did you restart the service
> after changing the config?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tananaev/traccar/issues/1574#issuecomment-163739128>.
>



-- 
CIS Technologies (P) Ltd.

**Comment by cispune** [comment]
is you are storing user side (Web UI) data in localStorage on client pc?

On Fri, Dec 11, 2015 at 10:43 AM, CIS <cis.service.pune@gmail.com> wrote:

> yes restarted the service
>
> On Fri, Dec 11, 2015 at 1:52 AM, Anton Tananaev <notifications@github.com>
> wrote:
>
>> I guess Traccar is still using old database. Did you restart the service
>> after changing the config?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/tananaev/traccar/issues/1574#issuecomment-163739128>.
>>
>
>
>
> --
> CIS Technologies (P) Ltd.
>



-- 
CIS Technologies (P) Ltd.

**Comment by tananaev** [comment]
No, Traccar doesn't use web local storage. All data is kept in the database on the server side.
